odoo/models.py

```diff
@@ -4349,18 +4349,21 @@ def _check_company(self, fnames=None):
             lines = [_("Incompatible companies on records:")]
             company_msg = _lt("- Record is company “%(company)s” and “%(field)s” (%(fname)s: %(values)s) belongs to another company.")
             record_msg = _lt("- “%(record)s” belongs to company “%(company)s” and “%(field)s” (%(fname)s: %(values)s) belongs to another company.")
+            record_companies_msg = _lt("- “%(record)s” belongs to companies %(company)s and “%(field)s” (%(fname)s: %(values)s) belongs to another company.")
```

Comment:
the message may have some English grammar issues
it would be 
`- record_a belongs to companies company_a, company_b and field_a belongs to another company.`
the `and` can be used to describe the combination among `comapny_a, company_b and field_a` which is confusing

Reply:
I'll replace `and` by `but`, how does that sound to you? :smile: